ios/engine/build.sh

```diff
@@ -0,0 +1,168 @@
+#!/usr/bin/env bash
+
+## START STANDARD BUILD SCRIPT INCLUDE
+# adjust relative paths as necessary
+THIS_SCRIPT="$(readlink -f "${BASH_SOURCE[0]}")"
+. "${THIS_SCRIPT%/*/*}/../resources/build/build-utils.sh"
+## END STANDARD BUILD SCRIPT INCLUDE
+
+# This script runs from its own folder
+cd "$(dirname "$THIS_SCRIPT")"
+
+# Include our resource functions; they're pretty useful!
+. "$KEYMAN_ROOT/resources/shellHelperFunctions.sh"
+. "$KEYMAN_ROOT/resources/build/build-download-resources.sh"
+
+# Please note that this build script (understandably) assumes that it is running on Mac OS X.
+verify_on_mac
+
+builder_describe "Builds Keyman Engine for use on iOS devices - iPhone and iPad." \
+  "@/web/src/app/webview        build" \
+  "@/common/web/sentry-manager  build" \
+  "clean" \
+  "configure" \
+  "build" \
+  "--debug         Avoids codesigning and adds full sourcemaps for the embedded predictive-text engine" \
+  "--sim-artifact  Also outputs a simulator-friendly test artifact corresponding to the build"
+
+builder_parse "$@"
+
+CONFIG="release"
+if builder_is_debug_build; then
+  CONFIG="debug"
+fi
+
+builder_describe_outputs \
+  configure     /ios/Carthage/Build     \
+  build         /ios/build/Build/Products/Release/KeymanEngine.xcframework
+
+# Base definitions (must be before do_clean call)
+DERIVED_DATA="$KEYMAN_ROOT/ios/build"
+BUILD_PATH=$DERIVED_DATA/Build/Products
+
+# Extended path definitions
+KMEI_RESOURCES=engine/KMEI/KeymanEngine/resources
+BUNDLE_PATH=$KMEI_RESOURCES/Keyman.bundle/Contents/Resources
```

Comment:
```suggestion
BUNDLE_PATH="${THIS_SCRIPT_PATH}/KMEI/KeymanEngine/resources/Keyman.bundle/Contents/Resources"
```

And then eliminate `KMEI_RESOURCES` variable which is only used here anyway?